include/pnet_api.h

```diff
@@ -1148,6 +1148,8 @@ typedef struct pnet_ip_addr_t
 
 /**
  * The Ethernet MAC address.
+ *
+ * See also pnal_ethaddr_t
```

Comment:
I have not really understood the relationship between these type defs. Can one of them be removed?

Reply:
Is there an elegant solution @hefloryd ?

Reply:
I don't see why there are two types for this. Possibly left-over from previous refactorings. Probably pnal_ethaddr_t should be removed?

Reply:
I will change it to a TODO in this documentation update.

Reply:
Done